src/Knp/Snappy/AbstractGenerator.php

```diff
@@ -516,7 +517,11 @@ protected function isAssociativeArray(array $array)
      */
     protected function executeCommand($command)
     {
-        $process = new Process($command, null, $this->env);
+        if (Kernel::VERSION_ID >= 40200) {
```

Comment:
You can't use `Symfony\Component\HttpKernel\Kernel::VERSION_ID` because `symfony/process` can be installed in version 4.2 but `symfony/http-kernel` in version 4.1.